pootle/apps/pootle_fs/management/commands/init_fs_project.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import logging
+
+from django.core.management import BaseCommand, CommandError
+
+from pootle_project.models import Project
+from pootle_language.models import Language
+
+
+logger = logging.getLogger('pootle.fs')
+
+
+class Command(BaseCommand):
+    help = "Init a new Pootle FS project."
+
+    def add_arguments(self, parser):
+        super(Command, self).add_arguments(parser)
+        parser.add_argument(
+            '--code',
+            action='store',
+            dest='code',
+            help='Project code'
+        )
+        parser.add_argument(
+            '--name',
+            action='store',
+            dest='name',
+            nargs='?',
+            help='Project name'
+        )
+        parser.add_argument(
+            '--fs',
+            action='store',
+            help='FS url'
+        )
+        parser.add_argument(
+            '--translation-path',
+            action='store',
+            dest='translation_path',
+            help='Translation path',
+            nargs='?',
+            default='<language_code>/<dirpath>/<filename>.<ext>'
+        )
+        parser.add_argument(
+            '--file-type',
+            action='store',
+            dest='file_type',
+            help='File type',
+            nargs='?',
+            default='po'
+        )
+        parser.add_argument(
+            '--checkstyle',
+            action='store',
+            dest='checkstyle',
+            help='Checkstyle',
+            nargs='?',
+            default='standard'
+        )
+        parser.add_argument(
+            '-l --source_language',
+            action='store',
+            dest='source_language',
+            help='Source language code',
+            nargs='?',
+            default='en'
```

Comment:
I don't feel it is right to hardcode here defaults. Better check below if some value was provided, and if that is the case pass it to project creation call.